examples/spark/src/main/java/org/apache/spark/streaming/receiver/example/ProducerSparkReceiverData.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.spark.streaming.receiver.example;
+
+import java.nio.charset.Charset;
+import org.apache.pulsar.client.api.Producer;
+import org.apache.pulsar.client.api.PulsarClient;
+
+/**
+ * producer data to spark streaming receiver.
+ *
+ * pulsar://localhost:6650 test_src
+ */
+public class ProducerSparkReceiverData {
+
+  public static void main(String[] args) throws Exception {
+    if (args.length < 2) {
+      System.out.println("Missing parameters!");
+      System.out.println("Usage: <pulsar-service-url> <topic>");
+      return;
+    }
+
+    System.out.println("Parameters:");
+    System.out.println("\tServiceUrl:\t" + args[0]);
+    System.out.println("\tTopic:\t" + args[1]);
+
+    PulsarClient client = PulsarClient.builder().serviceUrl(args[0]).build();
+    Producer<byte[]> producer = client.newProducer().topic(args[1]).create();
+
+    for (int i = 0; i < 100; i++) {
+       producer.send(("producer spark streaming msg").getBytes(Charset.defaultCharset()));
```

Comment:
nit: I would suggest using `UTF_8`. if your producer and consumer live in two different machines with different default charsets, the decoding will fail.